### DISCUSSION TITLE
Remove method `TSDataset.loc`

### DISCUSSION THREAD

**Discussion by d-a-bunin** [etna-team/etna#434] [Improvements]
### Description

Currently, we have a method `TSDataset.loc` which returns `pd.DataFrame` and gives write access to `TSDataset.df` attribute.

### Thoughts

We shouldn't give write access to `TSDataset.df`, because it could allow users to break it.

### Related

- #422.

**Comment by d-a-bunin** [comment]
Created issue #563

**Comment by egoriyaa** [comment]
Solved in #631
